frontend/src/SignUp.module.css

```diff
@@ -0,0 +1,57 @@
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    margin: 0; 
+    background-color: rgb(70, 188, 247); 
+    border: solid;
+}
+
+.form {
+    display: flex;
+    flex-direction: column;
+    max-width: 50%;
+    width: 150%; 
+    padding: 30%; 
+    background-color: rgb(255, 255, 255);
+    border-radius: 1em;
+    justify-content: center;
+    border: solid blue;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+.form label {
+    margin-bottom: 1em;
+    margin-top: 1em;
+    margin-right: auto;
+
+}
+
+.form input[type="submit"] {
+    text-align: center;
+    margin-top: 1.5em;
+    padding: 1em;
+    background-color: rgb(66, 172, 239);
+
+    
+}
+.form-group {
+    display: flex;
+    flex-direction: column;
+  }
+
+.logInButton {
```

Comment:
let's standardize on hyphenated class names like you have above for `.form-group`